File: Tugas coding modul 2/coding c/PRAK204-2310817120005-Noviana_Nur_Aisyah.c
#include <stdio.h>
#include <math.h>

#define pi  3.142857

int main(){

    double a, b;

    printf("Input\n");
    scanf("%lf %lf", &a, &b);

    double volume = pi * pow(a, 2) * b;
    double luas = 2 * pi * a * (a + b);
    double keliling = 2 * pi * a;

    printf("\nOutput\n");
    printf("Volume = %.2lf\n", volume);
    printf("Luas = %.2lf\n", luas);
    printf("Keliling = %.2lf\n", keliling);

    return 0;
}

File: Tugas coding modul 2/coding py/PRAK204-2310817120005-Noviana_Nur_Aisyah.py
print('Input')
input_aja = input() 
# jari-jari # tinggi bejana
# 7 10 | 10 10

angka = input_aja.split()

angka1, angka2 = map(int, angka)
# angka1 = jari-jari, angka2 = tinggi

phi = 22 / 7
v = phi * (angka1**2) * angka2
l = 2 * phi * angka1 * (angka1 + angka2)
k =  2 * phi * angka1

print('\nOutput')
print(f'Volume = {v:.2f}')
print(f'Luas = {l:.2f}')
print(f'Keliling = {k:.2f}')